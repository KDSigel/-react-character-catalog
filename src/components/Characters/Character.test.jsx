import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Character from "./Character";


it('should render the Character', () => {

const character = {
    image: 'someurl',
    name: 'fred',
    species: 'ape',
    status: 'dead',
}

    const { container } = render(
      <MemoryRouter>
        <Character character={character}/>
      </MemoryRouter>
    );
  
    expect(container).toMatchSnapshot();
  });