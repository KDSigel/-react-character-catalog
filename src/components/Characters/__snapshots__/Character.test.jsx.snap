// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Character 1`] = `
<div>
  <figure>
    <img
      alt="fred"
      src="someurl"
    />
    <figcaption>
      <h3>
        fred
      </h3>
      <p>
        ape
      </p>
      <p>
        Status: 
        dead
      </p>
    </figcaption>
  </figure>
</div>
`;
